cmake/FindBullet.cmake

```diff
@@ -1,77 +0,0 @@
-# - Try to find the Bullet physics engine
```

Comment:
we used to have our own simulator, then we switched to grSim (now we use erforce's sim). Our 2012 and 2013 [tdps](https://wiki.robojackets.org/Team_Description_Papers) talk about it a good bit